maria-oss/src/index.ts

```
@@ -0,0 +1,11 @@
+/**
+ * MARIA - AI-Powered Development Platform
+ * OSS Distribution Entry Point
+ * @version 1.0.7
+ */
+
+// CLI entry point
+export { createCLI } from './cli';
+
+// Version
+export const VERSION = '1.0.7';
```

ESLint

[error] Insert `⏎`

(prettier/prettier)